scene/3d/skeleton.h

```diff
@@ -35,9 +35,12 @@
 #include "scene/3d/spatial.h"
 #include "scene/resources/skin.h"
 
-#ifndef _3D_DISABLED
+#ifndef BONE_ID_DEF
```

Comment:
What's the point of these guards? There are already header guards for the whole file so this code will only be included once.

Reply:
I think the intent is to expose the bone def globally. However, I can remove.

Reply:
But no existing code nor code added in this PR needs it, so it seems pointless?